docker-compose.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: '3'
```

Comment:
was there a specific reason for that?

Reply:
What is the reason for 3.7? I don't have it on machine and see no reason for that.

Reply:
It was changed on purpose last November. I think it makes sense to ask the person who changed it instead of silently lowering the version.

Reply:
@v-stepanov In regards to this I've spent some time figuring out what and why happen.
The change was introduced by means of this commit: https://github.com/zalando/nakadi/commit/fc0c7e82c1d194d136cdf0e125965d7ee5d1e62b of this pr: https://github.com/zalando/nakadi/pull/980/files
I took a look here - https://docs.docker.com/compose/compose-file/compose-versioning/#version-3
There are no properties, that we are using and that were added in versions 3.1-3.7. 

In general the comment is - we are not using any features from 3.7, but insisting on it. If you have newest installation ever - our docker-compose should work. But from the other hand - why should we force people to use this latest version? 

I feel that the answer is: just because we can.

And the last part - I we want to use latest docker-compose+docker installation - let's use it. But don't enforce it through file format version.